Refresh.GameServer/Types/Levels/GameLevel.cs

```diff
@@ -76,9 +76,14 @@ public GameLevelType LevelType
     // ReSharper disable once InconsistentNaming
     internal int _LevelType { get; set; }
 
-    [Ignored] public GameLevelSource Source
+    /// <summary>
+    /// The source slot type that a level has come from.
+    ///
+    /// The only valid values of this are `User` or `Story`.
```

Comment:
Should be in `<remarks>` or just left in xmldoc for `GameSlotType`. Or not at all, because this is already invalid with the addition of `playlist`.

Reply:
I think i'm going to replace this field with a boolean `IsStoryLevel` or something. How I originally wrote it when i added story levels doesn't make sense to me anymore. I don't know why i used the slot type here.

Reply:
Resolved this by removing `Source` entirely and replacing it with logic that does `if story id == 0 then user slot else story slot`. Slot ID 0 is invalid in LBP anyway, so there's no worry of it conflicting.